src/test/recovery/Makefile

```diff
@@ -0,0 +1,39 @@
+#-------------------------------------------------------------------------
+#
+# Makefile for src/test/recovery
+#
+# Losely based on the makefile found in postgres' src/test/recovery.
+# We need to define our own invocation of prove to pass the correct path
+# to pg_regress and include citus in the shared preload libraries.
+#
+#-------------------------------------------------------------------------
+
+subdir = src/test/recovery
+top_builddir = ../../..
+include $(top_builddir)/Makefile.global
+
+# copied from pgxs/Makefile.global to use postgres' abs build dir for pg_regress
+ifeq ($(enable_tap_tests),yes)
+
+define citus_prove_installcheck
+rm -rf '$(CURDIR)'/tmp_check
+$(MKDIR_P) '$(CURDIR)'/tmp_check
+cd $(srcdir) && \
+TESTDIR='$(CURDIR)' \
+PATH="$(bindir):$$PATH" \
+PGPORT='6$(DEF_PGPORT)' \
```

Comment:
what's the 6 for?

Reply:
It comes from https://github.com/postgres/postgres/blob/472e518a44eacd9caac7d618f1b6451672ca4481/src/Makefile.global.in#L451

This is to run the TAP tests on the non default port of 5432, instead it runs on 65432.